golem-service-base/src/storage/blob/s3.rs

```diff
@@ -124,6 +127,9 @@ impl S3BlobStorage {
                         .to_path_buf()
                 }
             }
+            BlobStorageNamespace::InitialComponentFiles => {
+                Path::new(&self.config.object_prefix).to_path_buf()
+            }
```

Comment:
Are we sure we have to move this from worker-executor-base to service-base? I can see the need, but in that case, probably to golem-common.

Reply:
It makes sense to share the storage abstractions among services, and it also makes sense to not put it in golem-common which is referenced by things like the generated client library and CLI as well. 

Yes we talked about earlier that we could move all these things into a single crate and use features to avoid having all these backend stuff on the client side, but I'm not sure it would be better and would not make the build flow even slower, so at the current state moving these to `golem-service-base` totally makes sense and is the best choice.